offload/liboffload/src/OffloadImpl.cpp

```diff
@@ -34,6 +34,69 @@ struct ol_platform_handle_t_ {
   std::vector<ol_device_handle_t_> Devices;
 };
 
+struct ol_queue_handle_t_ {
```

Comment:
I don't love the '_' suffix idea. It's like there is something missing.

Reply:
I've changed this to `ol_*_queue_impl_t`